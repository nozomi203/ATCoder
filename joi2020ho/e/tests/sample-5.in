20 20
2 1 2 2 1 1 1 1 2 2 2 1 2 1 1 2 1 2 1 1
1 1 14
2 3 18
4 10 15
8 2 17
9 20 20
4 8 19
7 2 20
11 1 5
13 2 8
20 1 20
2 12 15
7 1 14
12 7 18
14 2 17
9 19 20
12 12 12
6 2 15
11 2 15
19 12 17
4 1 20
