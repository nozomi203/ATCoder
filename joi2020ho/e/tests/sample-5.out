25
30
12
32
2
24
38
10
14
40
8
28
24
32
4
2
28
28
12
40
